assets/css/_setup/_layout.scss

```
@@ -171,7 +175,7 @@ h3 {
 .narrow {
 	@include large-or-more {
 		margin: 0 auto;
-		width:  84%;
+		width:  100%;
```

Peut-être plutôt enlever la classe dans le `twig` et laisser cette valeur :
```suggestion
		width:  84%;
```